### PR TITLE
Fix more typescript --strict violations

### PR DESCRIPTION
**Pull request by t3chguy**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Review by SimonBrandner — APPROVED** [review]
Otherwise LGTM

File: spec/unit/event.spec.ts [hunk 1]
```diff
@@ -1,65 +0,0 @@
-/*
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply spec/unit/event.spec.ts:1]: Moved the tests into spec/unit/models/event.spec.ts - that's the correct path for this unit

File: src/models/event-context.ts [hunk 2]
```diff
@@ -95,8 +95,8 @@ export class EventContext {
      * @param {boolean} backwards   true to set the pagination token for going
      *                                   backwards in time
      */
-    public setPaginateToken(token: string, backwards = false): void {
-        this.paginateTokens[backwards ? Direction.Backward : Direction.Forward] = token;
+    public setPaginateToken(token?: string, backwards = false): void {
+        this.paginateTokens[backwards ? Direction.Backward : Direction.Forward] = token ?? null;
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply src/models/event-context.ts:99]: The types forbade it, so I hope not

**Comment by t3chguy** [comment]
Bypassing coverage requirement due to the nature of this PR

**Pull request merged**